src/d3a/models/state.py

```diff
@@ -27,20 +27,6 @@
 
 StorageSettings = ConstSettings.StorageSettings
 
-# Complex device models should be split in three classes each:
```

Comment:
This comment is important to keep here IMO. We already violate this a lot by not assigning enough things to the state.

Reply:
Ok, keeping in mind our offline discussion i will re-add and slightly change this comment.